app/Http/Controllers/EventsController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Models\Events;
+
+class EventsController extends Controller
+{
+    public function byMonth(Request $request)
+    {
+        $year = $request->input('year');
+        $month = $request->input('month');
+        
+        $monthStart = mktime(0, 0, 0, $month, 1, $year);
+        $monthEnd = mktime(0, 0, 0, $month+1, 0, $year);
```

Comment:
What if `$month=12`? I think it just doesn't handle it this way. I think you should use `Carbon` to handle parsing date and adding months to it